src/main/java/task/mentorship/application/controller/UserController.java

```diff
@@ -0,0 +1,84 @@
+package task.mentorship.application.controller;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import task.mentorship.application.security.*;
+import task.mentorship.application.entity.*;
+import task.mentorship.application.exception.CustomException;
+import task.mentorship.application.dto.*;
+import task.mentorship.application.service.*;
+@RestController
+@RequestMapping("/api")
+public class UserController {
+    private final UserService userService;
+    private final AuthService authService;
+    private final JwtTokenProvider jwt;
+
+
+    public UserController(UserService userService, AuthService authService, JwtTokenProvider jwt) {
+		super();
+		this.userService = userService;
+		this.authService = authService;
+		this.jwt = jwt;
+	}
+    @CrossOrigin(origins = "http://localhost:4200")
```

Comment:
Let's move this URL "http://localhost:4200" to an env variable instead
and put @CrossOrigin over the whole control, not per each API